src/cli/subcommands/db_cmd.rs

```diff
@@ -106,6 +122,67 @@ impl DBCommands {
                     }
                 }
             }
+            DBCommands::Dump {
+                output_path,
+                no_compression,
+            } => {
+                println!("Dumping database to CAR file");
```

Comment:
I would refactor this into it's own method for readability. And some documentation would be much appreciated.